code/LINDA/LINDA_turf_tile.dm

```diff
@@ -274,11 +274,11 @@ atom/movable/var/pressure_resistance = 5
 atom/movable/var/last_forced_movement = 0
 
 atom/movable/proc/experience_pressure_difference(pressure_difference, direction)
-	if(last_forced_movement >= SSair.times_fired)
+	if(last_forced_movement >= world.time-10)

```

Comment:
From my understanding, 1 second.

This keeps fastmos from pulling people out of an air lock twice a second.

Is there a better way to do this?